freight_calibration/scripts/calibrate_freight

```diff
@@ -0,0 +1,206 @@
+    copy_file("/tmp/robot.launch", os.path.join(install_dir, "robot.launch"))
+
+def get_calibration_dict(directory, calibration = None):
+    if calibration == None:
+        calibration = dict()
+    # Load file
+    try:
+        # Get name of latest file
+        files = [f for f in os.listdir(directory) if f.startswith("calibration_") and f.endswith(".yaml")]
+        files.sort()
+        # Open it
+        calfile = open(os.path.join(directory, files[-1]))
+    except IndexError, IOError:
+        print("Cannot open calibration.yaml")
+        return calibration
+    # Parse YAML
+    try:
+        calyaml = yaml.load(calfile)
+    except:
+        print("Cannot parse calibration.yaml")
+        return calibration
+    # Get dictionary data
+    calibration["depth_camera_info_url"] = calyaml["depth_info"]
+    calibration["rgb_camera_info_url"] = calyaml["rgb_info"]
+    calibration["robot_description"] = calyaml["urdf"]
+    return calibration
+
+def get_base_calibration_dict(directory, calibration = None):
+    if calibration == None:
+        calibration = dict()
+    # Load file
+    try:
+        calfile = open(os.path.join(directory, "base_calibration.yaml"))
+    except IOError:
+        print("Cannot open base_calibration.yaml")
+        return calibration
+    # Parse YAML
+    try:
+        calyaml = yaml.load(calfile)
+    except:
+        print("Cannot parse base_calibration.yaml")
+        return calibration
+    # Get dictionary data
+    calibration["/base_controller/track_width"] = str(calyaml["odom"])
+    calibration["/imu/gyro_scale"] = str(calyaml["imu"])
+    return calibration
+
+def move_calibration(current_directory, new_directory, rosdistro):
+    calibration = get_calibration_dict(current_directory)
+    print( current_directory)
+    calibration = get_base_calibration_dict(current_directory, calibration)
+
+    # install urdf, depth, rgb files
+    for name in ["depth_camera_info_url", "rgb_camera_info_url", "robot_description"]:
+        try:
+            copy_file(os.path.join(current_directory, calibration[name]),
+                      os.path.join(new_directory, calibration[name]))
+        except:
+            print(current_directory)
+	    print(new_directory)            
+            print("Unable to copy %s" % name)
+
+    # update the robot.launch
+    update_robot_launch("/etc/ros/%s/robot.launch" % rosdistro, calibration, new_directory)
+
+def get_calibration_date(rosdistro):
+    launch_xml = etree.parse("/etc/ros/%s/robot.launch" % rosdistro)
+    for child in launch_xml.getroot():
+        if child.tag == "param" and child.get("name") == "calibration_date":
+            return child.get("value")
+    return "Calibration date not found in robot.launch"
+
+if __name__ == "__main__":
+    # Parse the arguments
+    parser = argparse.ArgumentParser(description="Calibrate the robot, update files in /etc/ros")
+    parser.add_argument("--base", help="Captyure base calibration data", action="store_true")
```

Comment:
also -- "base" is for the odometry/imu stuff -- it seems we are missing a "base_camera" or similar, to actually run capture.launch (as we do with the "arm" parameter in capture_robot in fetch_calibration